test/e2e/example.test.js

```diff
@@ -104,58 +89,42 @@ describe(`Example Tests`, function() {
       };
     });
 
-    // Add docsify target element
-    // https://playwright.dev/#path=docs%2Fapi.md&q=pageevalselector-pagefunction-arg
-    await page.$eval('body', elm => {
-      elm.innerHTML = '<div id="app"></div>';
-    });
-
     // Inject docsify theme (vue.css)
-    // https://playwright.dev/#path=docs%2Fapi.md&q=pageaddstyletagoptions
     await page.addStyleTag({ url: '/lib/themes/vue.css' });
 
     // Inject docsify.js
-    // https://playwright.dev/#path=docs%2Fapi.md&q=pageaddscripttagoptions
     await page.addScriptTag({ url: '/lib/docsify.js' });
 
     // Wait for docsify to initialize
-    // https://playwright.dev/#path=docs%2Fapi.md&q=pagewaitforselectorselector-options
     await page.waitForSelector('#main');
 
     // Create handle for JavaScript object in browser
-    // https://playwright.dev/#path=docs%2Fapi.md&q=pageevaluatepagefunction-arg
     const $docsify = await page.evaluate(() => window.$docsify);
+    // const $docsify = await page.evaluateHandle(() => window.$docsify);
 
     // Test object property and value
     expect($docsify).toHaveProperty('themeColor', 'red');
   });
 
-  test('Docsify /docs/ site using docsifyInit()', async () => {
+  test('Docsify /docs/ site using docsifyInit()', async ({ page }) => {
     // Load custom docsify
     // (See ./helpers/docsifyInit.js for details)
     await docsifyInit({
       config: {
         basePath: '/docs/',
       },
-      // _debug: true,
       // _logHTML: true,
     });
 
-    // Create handle for JavaScript object in browser
-    // https://playwright.dev/#path=docs%2Fapi.md&q=pageevaluatepagefunction-arg
-    const $docsify = await page.evaluate(() => window.$docsify);
-
-    // Verify config options
-    expect(typeof $docsify).toEqual('object');
-
     // Verify docsifyInitConfig.markdown content was rendered
-    await expect(page).toHaveText(
-      '#main',
+    const mainElm = page.locator('#main');
+    await expect(mainElm).toHaveCount(1);
+    await expect(mainElm).toContainText(
       'A magical documentation site generator'
     );
   });
 
-  test('custom docsify site using docsifyInit()', async () => {
+  test('custom docsify site using docsifyInit()', async ({ page }) => {
```

Comment:
Hard to say what the issue is in the scenario above, but I don't think it's an issue with a global `page` object because I'm still using a global page object in this update (which I'm mulling over the best way to get rid of).

Playwright has undergone significant development since we adopted in over a year ago, so we needed this update just to bring us on par with the project. The updated docs also specifically recommend against using the original "dollar sign" methods like `$`, `$$`, `$eval`, and `$$eval` which I was using in a few areas (per the instructions from the old docs). This PR follows the new recommendation to replace these methods with the newer [`locator`](https://playwright.dev/docs/api/class-page#page-locator) and [`evaluate`](https://playwright.dev/docs/api/class-page#page-evaluate) methods. Here's are two screenshots from the docs:

<img width="794" alt="CleanShot 2022-01-28 at 21 54 19@2x" src="https://user-images.githubusercontent.com/442527/151648334-2137886f-595f-4862-8e2c-87f6ec4382d1.png">

<img width="793" alt="CleanShot 2022-01-28 at 21 54 38@2x" src="https://user-images.githubusercontent.com/442527/151648344-f0379c95-666e-47d6-98c1-285bdf59e77f.png">

Between the "flakey" method swap, the new Playwright runner, and a year's worth of development, I'm hoping this PR will address and test stability issues people have been dealing with.